### DISCUSSION TITLE
State Timeline Panel - Hide y-axis option

### DISCUSSION THREAD

**Discussion by BariscanE** [grafana/grafana#38847] [Dashboards, Panels & Visualizations]
In order to correlate stacked state timeline panels, we need the hide y-axis option. 

By default each of the y-axis (labels) are in different size and when you stack multiple state timeline panels on top of each other, timelines do not match hence you cannot correlate the metrics. With hide y-axis option available, that will not be the case since each state timeline panels will have the same size. Below is an example:

![resim](https://user-images.githubusercontent.com/75362049/131958022-f451230b-c231-4048-8fbf-4316fbebcaa0.png)

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
